### PR TITLE
Add the milestone to velodrome

### PR DESCRIPTION
**Pull request by grodrigues3**
Add the issue milestone to velodrome, so we can look at issues/PRs in a milestone

### CODE REVIEW

File: velodrome/fetcher/conversion.go [hunk 1]
```diff
@@ -73,6 +73,7 @@ func NewIssue(gIssue *github.Issue, repository string) (*sql.Issue, error) {
 		IssueClosedAt:  closedAt,
 		IssueCreatedAt: *gIssue.CreatedAt,
 		IssueUpdatedAt: *gIssue.UpdatedAt,
+		Milestone: *gIssue.Milestone,
```
Comment (apelisse) [inline velodrome/fetcher/conversion.go]: Mis-aligned?

**Comment by grodrigues3** [comment]
@spxtr any idea what's going on the bazel build here:

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/2635/pull-test-infra-bazel/2443/

**Comment by spxtr** [comment]
`I0502 19:27:04.778] k8s.io/test-infra/velodrome/fetcher/conversion.go:76: cannot use *gIssue.Milestone (type github.Milestone) as type string in field value`

File: velodrome/fetcher/conversion.go [hunk 2]
```diff
@@ -73,6 +73,7 @@ func NewIssue(gIssue *github.Issue, repository string) (*sql.Issue, error) {
 		IssueClosedAt:  closedAt,
 		IssueCreatedAt: *gIssue.CreatedAt,
 		IssueUpdatedAt: *gIssue.UpdatedAt,
+		Milestone_ID:   strconv.Itoa(*gIssue.Milestone.ID),
```
Comment (apelisse) [inline velodrome/fetcher/conversion.go]: Gorm is looking for a specific format. It should probably be `MilestoneID`

File: velodrome/sql/model.go [hunk 3]
```diff
@@ -32,6 +32,7 @@ type Issue struct {
 	User           string
 	State          string
 	Comments       int
+	Milestone_ID   string
```
Comment (apelisse) [inline velodrome/sql/model.go]: Same as above `MilestoneID`

File: velodrome/sql/model.go [hunk 4]
```diff
@@ -89,3 +90,15 @@ type Comment struct {
 	CommentUpdatedAt time.Time
 	PullRequest      bool
 }
+
+// Milestone contains information about github milestons and can be added to issues and prs
+type Milestone struct {
```
Comment (apelisse) [inline velodrome/sql/model.go]: Do we need all these fields?

Also do you expect this structure to be serialized? We don't have json annotation for other structures.

**Comment by apelisse** [comment]
I would also be interested in knowing how this is going to be populated (especially for already existing issues).

**Comment by fejta** [comment]
Is this still alive? Please update and/or close

**Comment by grodrigues3** [comment]
I forgot about this.  I'll look into it more this week.

**Comment by grodrigues3** [comment]
/close

I'm not sure if this is relevant anymore.

**Pull request closed without merging**